build.sbt

```diff
@@ -31,7 +31,8 @@ lazy val microservice = Project(appName, file("."))
     libraryDependencies ++= AppDependencies.compile ++ AppDependencies.test(),
     retrieveManaged := true,
     update / evictionWarningOptions := EvictionWarningOptions.default.withWarnScalaVersionEviction(warnScalaVersionEviction = false),
-    scalaVersion := "2.12.15"
+    scalaVersion := "2.12.15",
+    scalacOptions ++= Seq("-Xfatal-warnings", "-Wconf:src=routes/.*:silent", "-feature", "-language:higherKinds")
```

Comment:
The "-Wconf:src=routes/.*:silent" setting is doing the same work as the silencer plugin, so you can delete the 
* ".settings(SilencerSettings())" line in here
* as well as the SilencerSettings.scala file.